client/src/components/lower-info-area/new-and-better-way.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <v-container id="new-and-better-way" class="pt-16">
+    <v-row no-gutters class="pa-9">
+      <v-col cols="6" class="pl-3 pr-75">
+        <v-img src="../../assets/images/bcnamerequest-laptop_x2.jpg" />
+      </v-col>
+      <v-col cols="6" class="pr-3 pl-75">
+        <v-row>
+          <v-col cols="12" class="h3 pb-5 mt-n5">
+            A new and better way to get a name <br/>for your business in B.C.
+          </v-col>
+          <v-col cols="12">
+            <ul>
+              <li><span>Find out if you can have your name right away (<i>Coming soon</i>)</span></li>
```

Comment:
Why do you need the span element?

Reply:
bullet point has to be in blue and text in black. to do that we need a separate element